qiskit/execute.py

```diff
@@ -158,6 +161,23 @@ def execute_schedules(schedules, backend, **kwargs):
     Returns:
         BaseJob: returns job instance derived from BaseJob
     """
+    if isinstance(schedules, Schedule):
+        schedules = [schedules]
+
+    if schedule_configs:
+        if isinstance(schedule_configs, ScheduleConfig):
+            schedule_configs = [schedule_configs]
+        if len(schedules) == len(schedule_configs):
```

Comment:
Can this be like
```
for schedule_config in schedule_configs:
    for schedule in schedules:
        experiments = ConfiguredSchedule(schedule, schedule_config)
```
In this way users can easily generate a recipe of spectrum mapping.

Reply:
for n:n case, we can directly provide `ConfiguredSchedule`.